### ISSUE TITLE
[Feature]: DIS/Is-Net background removal

### ISSUE THREAD

**Issue by Centurion-Rome** [vladmandic/automatic#1110]
### Feature description

It is very promissing method to remove background (IMO better as other one that I have seen) it would be great to integrate it somehow natively so we could generate transparent PNG (or on white background) automatically in SD.Next :)
https://github.com/xuebinqin/DIS

Some examples:
![peacock](https://github.com/vladmandic/automatic/assets/75789136/c2417dc1-4bff-4fe2-8157-6abe9cb4ae93)
![qual-comp](https://github.com/vladmandic/automatic/assets/75789136/b8227ded-dfdd-4b21-ae95-a8741f0b487c)

**Comment by vladmandic** [comment]
built-in one right now is not bad (`extensions-builtin/stable-diffusion-webui-rembg`), but this looks pretty amazing.
the implementation in repo is straightforward - i'd love if someone did this as an extension and then we can talk about making it a built-in one.

**Comment by mark-wd** [comment]
Excited about this prospect (although as vlad said, rembg does a pretty good job already). Hopefully it becomes and extension soon.

**Comment by mark-wd** [comment]
> It's not released yet. [xuebinqin/DIS#44 (comment)](https://github.com/xuebinqin/DIS/issues/44#issuecomment-1477270247)
> 
> The earlier version of the model (isnet-general-use) is already supported by the lib the extension uses (rembg), the update to support it is just waiting on being merged into the extension. [AUTOMATIC1111/stable-diffusion-webui-rembg#11](https://github.com/AUTOMATIC1111/stable-diffusion-webui-rembg/pull/11) [AUTOMATIC1111/stable-diffusion-webui-rembg#19](https://github.com/AUTOMATIC1111/stable-diffusion-webui-rembg/pull/19)

Thanks for the useful update.

**Comment by vladmandic** [comment]
ready in dev branch.
